scalafix-core/src/main/scala/scalafix/v1/SyntacticDocument.scala

```diff
@@ -29,15 +30,27 @@ final class SyntacticDocument private[scalafix] (
 }
 
 object SyntacticDocument {
-  @deprecated("use fromInput(input: Input, dialect: Dialect) instead", "0.9.28")
+  @deprecated(
+    "use fromInput(input: Input, scalaVersion: ScalaVersion) instead",
+    "0.9.28"
+  )
   def fromInput(input: Input): SyntacticDocument = {
-    fromInput(input, scala.meta.dialects.Scala212)
+    fromInput(input, ScalaVersion.scala2)
   }
-  def fromInput(input: Input, dialect: Dialect): SyntacticDocument = {
+  @deprecated(
+    "use fromInput(input: Input, scalaVersion: ScalaVersion) instead",
+    "0.9.28"
+  )
+  def fromInput(inout: Input, dialect: Dialect): SyntacticDocument = {
+    val tree = parse(inout, dialect).get: Tree
+    fromTree(tree)
+  }
```

Comment:
you can remove this since it was never published, no?

Reply:
This method was there too!